app/lib/populateCache.js

```diff
@@ -5,11 +5,12 @@ const assert = require('assert');
 
 function populateCacheSync(path) {
   const data = fs.readFileSync(path);
-  assert.ok(data.length > 0, `No data in file: '${path}'.`);
+  assert(data.length > 0, `No data in file: '${path}'.`);
 
   const records = parse(data, { columns: true });
```

Comment:
Should the code be explicit about what columns it expects and only use those columns in the cache.

Any thoughts?